style.css

```diff
@@ -0,0 +1,142 @@
+
+@import url('https://fonts.googleapis.com/css2?family=Dongle&family=Open+Sans:ital@1&family=Source+Sans+Pro:ital,wght@0,700;1,400&display=swap');
+@import url("./color-palette/color.css");
+
+body{
+    margin:auto !important;
+    padding:auto !important;
+}
+
+.tab-container{
+    display: flex;
+    width:100%;
+    height: 10vh;
+    background-color: var(--primary-color);
+    justify-content: flex-end;
+    position: fixed;
+
+}
+
+.tabContainer-contents{
+    padding: 0.4rem 1rem;
+    margin-top: 0;
+    margin-left:auto;
+
+}
+
+.tab-content{
+    margin: 1rem;
+    border: none;
+    background-color:transparent;
+    font-size: 1.25rem;
+    padding: 0.5rem;
+    color: var(--white);
+    cursor: pointer;
+}
+
+.tab-content:hover{
+    color: var(--secondary-color);
+}
+
+.tab-header-content > h1 {
+    color: var(--white);
+    position: absolute;
+    left: 20px;
+}
+
+
+.tab-header > h1:hover {
+    color: var(--secondary-color);
+}
+
+
+.landingpage-contents{
+    display:flex;
+    flex-flow: column wrap;
+    
+    position: absolute;
+    justify-content: center;
+    align-items: center;
+    top: 100px;
+}
+
+.landingpage-contents > img{
+    width: 31.25rem;
+    height: 15.625rem;
+    border-radius: 20%;
+    
+}
+.landingpage-contents > p {
+    position: relative;
+    padding:20px;
+    margin-left: 50px;
+    margin-right: auto;
+    font-weight: 600;
+    font-size:20px;
+    color: var(--primary-color);
+
```

Comment:
avoid using tag itself as selector , instead use classes

Reply:
I have changed that  thanks for reviewing